pandas/tests/arrays/string_/test_string.py

```diff
@@ -331,17 +481,23 @@ def test_use_inf_as_na(values, expected):
         tm.assert_frame_equal(result, expected)
 
 
-def test_memory_usage():
+def test_memory_usage(dtype, request):
     # GH 33963
-    series = pd.Series(["a", "b", "c"], dtype="string")
+
+    if dtype == "arrow_string":
+        reason = "assert 147 < 147"
+        mark = pytest.mark.xfail(reason=reason)
+        request.node.add_marker(mark)
```

Comment:
I don't think you need to parametrize this test, as it is specifically to the object-dtype based string dtype (or at least simply skip it, instead of an xfail)